flowtools/flowclr_sample_name/sample_simp.R

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/Rscript
+# Module for Galaxy
+# Adds sample information to a flowclustered file and binds 
+#
+#                  
+# Version 1
+# Emily Combe
+#
+#
+
+
+
+#-------------#
+# sample name #
+#-------------#
+
+
+library(flowCore)    
+
+sample_name <- function(input_file, sample_names){
+	flowtext <- list()
+	for(i in input_file){
```

Comment:
It seems to me that the tool works on a per file basis (so one input file, one output file, in other words, it is not merging files). If that is the case, then input_file should be a single file, not a set of files.